libraries/Microsoft.Bot.Builder.Dialogs.Declarative/Resources/FolderResourceExplorer.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.Bot.Builder.Dialogs.Declarative.Resources
+{
+    /// <summary>
+    /// An <see cref="ResourceExplorer"/> implementation that uses a <see cref="FolderResourceProvider"/>.
+    /// </summary>
+    public class FolderResourceExplorer : ResourceExplorer
+    {
+        private const string ApplicationRootKey = "applicationRoot";
```

Comment:
Does this really belong here? Seems like a asp.net core thing

Reply:
Let me check this one again - the other constants were resolved by moving the configuration head into the Integration.Runtime assembly. Perhaps something like that would help here.

Reply:
The runtime service collection extensions already do some effort to get the application root and also get the root folder. Check this out: https://github.com/microsoft/botbuilder-dotnet/blob/main/libraries/integration/Microsoft.Bot.Builder.Integration.Runtime/Extensions/ServiceCollectionExtensions.cs#L75

Reply:
Not sure we need this class at all tbh.

Reply:
I'm trying to get rid of 95% of the startup code - in this case by introducing a class
there also seems to be many nearly meaningless extension functions that are not contributing to the maintainability of this code

Reply:
If we keep this, we should add it to the class documentation (or the constructor) so devs know under what setting they should store appllicationRoot.